package.json

```
@@ -2,17 +2,18 @@
   "name": "react-colorful",
   "version": "2.3.1",
   "description": "A tiny color picker component for modern React apps",
-  "source": "src/hex.tsx",
+  "source": "src/index.ts",
   "main": "dist/index.js",
   "module": "dist/index.module.js",
   "esmodule": "dist/index.esmodule.js",
   "umd:main": "dist/index.umd.js",
-  "types": "dist/hex.d.ts",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
```

I believe we need to label the CSS as a side-effect

```suggestion
  "sideEffects": [
    "*.css"
  ],
```

---

Actually no. Sorry. Was thinking of some other situation in which I had to list the CSS as a side effect but that shouldn't be applicable here.

---

Hey @RyanChristian4427!
Just found out that you were right.
I added our picker to https://omatsuri.app and webpack removed all our styles from there.
See:
https://github.com/webpack/webpack/issues/8814#issuecomment-465223178

Released 3.0.2 patch with `"sideEffects": [ "*.css" ]` and now everything is fine.